lib/singlepass-backend/src/codegen_x64.rs

```diff
@@ -1114,6 +1233,29 @@ impl X64FunctionCode {
                     call_movs.push((*param, x));
                 }
                 Location::Memory(_, _) => {
+                    match *param {
+                        Location::GPR(x) => {
+                            let content = m.state.register_values[X64Register::GPR(x).to_index().0];
+                            //assert!(content != MachineValue::Undefined);
```

Comment:
Assertions needed or should they be removed?

Reply:
`debug_assert!` is also a good choice if you think you don't need it but future changes could cause it to be triggered

Reply:
Added some comments to explain why this assertion should remain commented.